### PR TITLE
feat: Support compute for multilang and/or multiple bones

### PR DESCRIPTION
**Pull request by sveneberth**

### CODE REVIEW

File: src/viur/core/bones/base.py [hunk 1]
```diff
@@ -1212,8 +1212,21 @@ def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', bone_name: str):
 
         ret = self.compute.fn(**compute_fn_args)
 
+        def unserialize_raw_value(value: list[dict] | dict | None):
+            if self.multiple:
+                return [self.singleValueUnserialize(value)
+                        for value in ret]
+            else:
+                return self.singleValueUnserialize(ret)
```
Comment (phorward) [inline src/viur/core/bones/base.py:1220]:
```suggestion
            return self.singleValueUnserialize(ret)
```

File: src/viur/core/bones/base.py [hunk 2]
```diff
@@ -1212,8 +1212,21 @@ def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', bone_name: str):
...
+
         if self.compute.raw:
-            return self.singleValueUnserialize(ret)
+            if self.languages:
+                return {
+                    lang: unserialize_raw_value(ret.get(lang, [] if self.multiple else None))
+                    for lang in self.languages
+                }
+            else:
+                return unserialize_raw_value(ret)
```
Comment (phorward) [inline src/viur/core/bones/base.py:1229]:
```suggestion
            return unserialize_raw_value(ret)
```

**Review by phorward — CHANGES_REQUESTED** [review]

File: src/viur/core/bones/base.py (hunk 1 above)
Comment (sveneberth) [reply src/viur/core/bones/base.py:1220]: To quote the second zen of python:
> Explicit is better than implicit.

But if the maintainer wants it that way, that's fine for me. :smiley:

Comment (phorward) [reply src/viur/core/bones/base.py]: Keep it short and simple. <3

**Review by phorward — APPROVED** [review]

**Pull request merged**